src/components/logs-query-input.tsx

```diff
@@ -48,13 +34,13 @@ export const LogsQueryInput: React.FC<LogsQueryInputProps> = ({
       className="co-logs-expression-input"
       data-test={TestIds.LogsQueryInput}
     >
-      <SearchInput
+      <TextArea
         className="co-logs-expression-input__searchInput"
         placeholder="LogQL Query"
-        value={searchValue}
+        value={value}
         onChange={handleSearchChange}
-        onClear={handleSearchClear}
         onKeyDown={handleKeyDown}
+        aria-label="LogQL Query"
       />
       {onRun && (
         <ExecuteQueryButton
```

Comment:
We could just pass `onClick={onRun}` here, removing the need for `handleRunQuery`